app/(app)/zone-details.tsx

```diff
@@ -57,11 +63,20 @@ const ZoneDetailsScreen = () => {
           </FlexRow>
         ) : null}
         {/* eslint-disable-next-line eqeqeq */}
+        {isAllWeekPrice || zone.price == undefined ? null : (
```

Comment:
also here... if you want to check if its null or undefined create a function or add there both undefined and null with ===

Reply:
changed to `== null` since this is also allowed by eslint

Reply:
Last time we discussed this in the office, we agreed that `== null` is hardly readable a prone to bugs, so it's better to use `isDefined` function, that we have in almost all projects.
This is just a note for future :)